crates/rolldown/tests/rolldown/topics/hmr/runtime_correctness/artifacts.snap

```diff
@@ -0,0 +1,251 @@
+// HIDDEN [rolldown:hmr]
+//#region cases/require/cjs-lib.js
+var require_cjs_lib = /* @__PURE__ */ __commonJS({ "cases/require/cjs-lib.js": ((exports, module) => {
+	const cjs_lib_hot = __rolldown_runtime__.createModuleHotContext("cases/require/cjs-lib.js");
+	__rolldown_runtime__.registerModule("cases/require/cjs-lib.js", module);
+	module.exports.foo = "foo";
+}) });
+
+//#endregion
+//#region cases/require/index.js
+var require_exports = {};
+const require_hot = __rolldown_runtime__.createModuleHotContext("cases/require/index.js");
+__rolldown_runtime__.registerModule("cases/require/index.js", { exports: require_exports });
+const requiredCjsLib = require_cjs_lib();
+assert.strictEqual(requiredCjsLib.foo, "foo");
+
+//#endregion
+//#region main.js
+var main_exports = {};
+const main_hot = __rolldown_runtime__.createModuleHotContext("main.js");
+__rolldown_runtime__.registerModule("main.js", { exports: main_exports });
+if (main_hot) {
+	main_hot.accept();
+}
+
+//#endregion
+```
+# HMR Step 0
+
+## Code
+
+```js
+//#region cases/require/cjs-lib.js
+var require_cjs_lib_0 = __rolldown_runtime__.createCjsInitializer((function(exports, module) {
+	try {
+		__rolldown_runtime__.registerModule("cases/require/cjs-lib.js", module);
+		const hot_cjs_lib = __rolldown_runtime__.createModuleHotContext("cases/require/cjs-lib.js");
+		module.exports.foo = "foo";
+	} finally {}
+}));
+
+//#endregion
+//#region cases/require/index.js
+import * as import_node_assert_0 from "node:assert";
+var init_require_1 = __rolldown_runtime__.createEsmInitializer((function() {
+	try {
+		var __rolldown_exports__ = {};
+		__rolldown_runtime__.__export(__rolldown_exports__, {});
+		__rolldown_runtime__.registerModule("cases/require/index.js", { exports: __rolldown_exports__ });
+		const hot_require = __rolldown_runtime__.createModuleHotContext("cases/require/index.js");
+		const requiredCjsLib = (require_cjs_lib_0(), __rolldown_runtime__.loadExports("cases/require/cjs-lib.js"));
+		import_node_assert_0.default.strictEqual(requiredCjsLib.foo, "foo");
+	} finally {}
+}));
+
+//#endregion
+//#region main.js
+var init_main_2 = __rolldown_runtime__.createEsmInitializer((function() {
+	try {
+		var __rolldown_exports__ = {};
+		__rolldown_runtime__.__export(__rolldown_exports__, {});
+		__rolldown_runtime__.registerModule("main.js", { exports: __rolldown_exports__ });
+		init_require_1();
+		const hot_main = __rolldown_runtime__.createModuleHotContext("main.js");
+		var import_require_0 = __rolldown_runtime__.loadExports("cases/require/index.js");
+		if (hot_main) {
+			hot_main.accept();
+		}
+	} finally {}
+}));
+
+//#endregion
+//#region rolldown:hmr
+var init_rolldown_hmr_3 = __rolldown_runtime__.createEsmInitializer((function() {
+	try {
+		var __rolldown_exports__ = {};
+		__rolldown_runtime__.__export(__rolldown_exports__, { DevRuntime: () => DevRuntime });
+		__rolldown_runtime__.registerModule("rolldown:hmr", { exports: __rolldown_exports__ });
```

Comment:
It seems the runtime is not hidden from the snapshot.

Reply:
I'll check